platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -113,12 +115,17 @@ public void onReceived(ObservedPayment payment) {
       txn.setStellarTransactionId(payment.getTransactionHash());
       try {
         transactionStore.save(txn);
+        Metrics.counter(
+                "sep31.transaction", "status", SepTransactionStatus.PENDING_RECEIVER.toString())
+            .increment();
```

Comment:
Hum, I don't think this solves the issue of avoiding string literals.

The problem with string literals is that we're humans and can't be sure if they're all correct. For instance, even though we've both reviewed your code we can't be sure there are no typos causing events to be generated into the wrong metric groups. You could easily have two calls like the following and not notice the issue:
* `Metrics.counter("logger", "type", "error").increment();`
* `Metrics.counter("logger", "typo", "error").increment();  // used "typo" instead of "type"`

A more appropriate approach here would be to have this enum in a package that's common to both core and platform. Why not define this enum inside `core` itself? This should make sense since you are already calling `Metrics.counter` from core, right?